templates/patterns/card/pattern-card--variant-search.html.twig

```diff
@@ -48,6 +48,13 @@
   {% set _right_col_classes = 'col-md-9 col-lg-10' %}
 {% endif %}
 
+{% set _date = date|default({}) %}
+{% if _date is not empty %}
+  {% set _date = _date|merge({
+    icon_path: bcl_icon_path
+  })%}
+{% endif %}
+
```

Comment:
```suggestion
{% set _date = date|default({})|merge({ icon_path: bcl_icon_path }) %}

```
I would suggest with the simpler solution and add the icon_path param in any case.

Reply:
Ok it makes sens, let's keep your code.